manager/integration/tests/test_basic.py

```diff
@@ -4558,3 +4561,66 @@ def test_backup_failed_disable_auto_cleanup(set_random_backupstore,  # NOQA
                       + volume_name + " is deleted"
     except AssertionError:
         pass
+
+
+def test_backup_save_access_mode(client, core_api, apps_api,  # NOQA
+                                 make_deployment_with_pvc):  # NOQA
+    """
+    Test the backup should be save access mode and restore based on the value
+
+    1. Create a volume with 'accessMode' rwx.
+    2. Create the backup for the volume.
+    3. Restore the backup without specific access mode
+    4. Check the restored volume's access mode should be 'readwritemany'
+    5. Restoring the backup using the `readwriteonce` access mode
+    6. Check the restored volume's access mode should be 'readwriteonce'
+    """
+
+    test_volume_name = "test-rwx-volume"
+    test_pv_name = test_volume_name + "-pv"
+    test_pvc_name = test_volume_name + "-pvc"
+    volume_access_mode = "rwx"
+
+    # create a volume
```

Comment:
Suggest adding the corresponding step number above, so it will be easier to understand which part is.

Reply:
Got it, I will add the step number to mark the section and the description will be in the head section.